src/components/Select.js

```diff
@@ -212,7 +212,7 @@ const styles = {
     position: 'relative'
   },
   downArrow: {
-    color: '#999999',
+    color: '#000000',
```

Comment:
No, so we'll need to merge the prop color onto the down arrow element here: https://github.com/jmophoto/mx-react-components/blob/jmo/select-icon-color/src/components/Select.js#L175. Something like this:
```
style={objectAssign(styles.downArrow, { color: this.props.selectedStyle.color || '#000000' })}
```